phpunit/fonts-library/class-wp-fonts-library-test.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Tests for Fonts Library class
+ *
+ * @package Gutenberg
+ */
+
+class WP_Fonts_Library_Test extends WP_UnitTestCase {
+
+	function test_get_fonts_directory() {
+		$this->assertTrue(
+			str_ends_with( WP_Fonts_Library::get_fonts_directory(), '/wp-content/fonts' )
```

Comment:
The [assertStringEndsWith()](https://docs.phpunit.de/en/9.6/assertions.html#assertstringendswith) assertion can be used in these tests.